mantaray/commands.py

```diff
@@ -154,6 +156,20 @@ def kick(view: View, core: IrcCore, nick: str, reason: str | None = None) -> Non
         else:
             view.add_message("You can use /kick only on a channel.", tag="error")
 
+    def away(view: View, core: IrcCore, away_message: str) -> None:
+        core.send(f"AWAY :{away_message}")
+        for view in view.server_view.get_subviews(include_server=True):
+            view.add_message("You have been marked as being away", tag="info")
```

Comment:
`tag="info"` is unnecessary, because the `tag` argument has `"info"` as its default value.